### DISCUSSION TITLE
classes/class_bitmap

### DISCUSSION THREAD

**Discussion by giscus** [godotengine/godot-docs-user-notes#29] [User-contributed notes]
# classes/class_bitmap

Inherits: Resource< RefCounted< Object Boolean matrix. Description: A two-dimensional array of boolean values, can be used to efficiently store a binary matrix (every matrix element takes only one ...

https://docs.godotengine.org/en/latest/classes/class_bitmap.html

**Comment by sanlor** [comment]
Its not in this doc, but there is a hidden "data" propriety in this class, very similar to the Image.data propriety. Very useful when you work with Multiplayer, since you cant send whole objects throught a Multiplayer peer, only certain variables (In this case, BitMap.data["data"]).
